priv/templates/phx.gen.auth/forgot_password_live_test.exs

```diff
@@ -12,8 +12,8 @@ defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web
       {:ok, _lv, html} = live(conn, ~p"<%= schema.route_prefix %>/reset_password")
 
       assert html =~ "Forgot your password?"
-      assert html =~ "Register</a>"
-      assert html =~ "Log in</a>"
```

Comment:
@aaronrenner + @josevalim Note that I've killed off the inner text in these assertions due to mess it'd have to make of the regexp matchers due to the additional whitespace added by the formatter. I'm comfortable with the removal of the text and closing-tag from these assertions due to their being covered in the snapshot tests in the injector tests, which do 1:1 string equality checks on the entirety of the output.